### DISCUSSION TITLE
Post processing LES results, best approach

### DISCUSSION THREAD

**Discussion by DonalMcC** [su2code/SU2#1907] [Q&A]
I have run an LES simulation in SU2. I've done 5 flow throughs as per rule of thumb, and am now trying to determine if my flow has reached or is approaching a statistically steady state.

I see SU2 has no feature to probe the flowfield while the simulation is running, and the developers have suggested outputting flowfiles and monitoring those. My issue with this approach is that my flow files are quite large (10gb) and if I want to look at several hundred timesteps, just to probe a few points in regions of interest, it's not a very efficient use of storage. I have tried this at the minute with 65 time steps but it wasn't enough to establish if a steady state was being approached.

I am considering modifying the code to ouput some properties at some nodes for each time step in the history file. Although, I'm not sure where to begin with this and given it's not an included feature, I think it might be trickier than I expect.

The other thing I was considering is using a python script with paraview or tecplot to open the flow files as they're written, probe the point, and then delete the flow file

Anyone else who has done LES in SU2, how have you handled this stage of the processing?
Has anyone taken the approaches I mentioned here?

**Comment by pcarruscag** [comment]
Can't you determine statistical convergence from the history of integrated quantities? Lift, drag, etc.

**Reply by DonalMcC** [reply]
I don't think so for two reasons: I'm using a wall model and I'm interested in the flow in the wake. I'm still new to LES but I think even if I wan't using a wall model, statisical convergence at the wall might not necessarily mean statistical convergence downstream. 

I think using an internal marker is the way to go, but as you mentioned here: https://github.com/su2code/SU2/issues/986, it may not be compatible with all aspects of SU2. I think I'm going to have to try it on a smaller case with similar settings
